converged_sound_open_fw_modules_development/FW/portable/include/modules.h

```diff
@@ -0,0 +1,15 @@
+  
```

Comment:
Is this file used to build FW or just the tools ? (as this will determine location)

Reply:
At the moment, IIRC ```MASTER_CORE_AFFINITY``` in ```modules.h``` is used only in binmaps, so it "participates" in creating manifest.

Reply:
ah, so it's to pre-process the binmaps to generate the output ?

Reply:
yes